ES README.md

```diff
@@ -0,0 +1,72 @@
+
+1, 1, 2, 5, 29, 866, ...
+
+Todos los elementos de la secuencia serán del campo finito (lo que significa que tanto el cuadrado como la adición se calculan módulo p).
+
+### Prueba STARK
+
+Crearemos una prueba para la afirmación "El elemento 1023<sup>rd</sup> de la secuencia FibonacciSq es...". Por "prueba" no nos referimos a una prueba matemática con deducciones lógicas, sino a algunos datos que puedan convencer a quien los lea de que la afirmación es correcta. Para hacerlo más formal, definimos dos entidades: el **Probador** y el **Verificador**. El Probador genera estos datos (proof). El Verificador recibe estos datos y verifica su validez. El requisito es que si la afirmación es falsa, el Probador no podrá generar una prueba válida (incluso si se desvía del protocolo).
+
+STARK es un protocolo específico que describe la estructura de tal prueba y define lo que el Probador y el Verificador tienen que hacer.
+
+### Algunas Otras Cosas que Debes Saber
+
+Le recomendamos que eche un vistazo a nuestras publicaciones matemáticas de [STARK math blog
+posts](https://medium.com/starkware/tagged/stark-math) (Arithmetization
+[I](https://medium.com/starkware/arithmetization-i-15c046390862) &
+[II](https://medium.com/starkware/arithmetization-ii-403c3b3f4355) specifically). No es necesario que las lea a fondo antes de seguir este tutorial, pero puede darle un mejor contexto sobre las cosas para las que puede crear pruebas y cómo se ve una prueba de STARK. Definitivamente debería leerlas después de completar este tutorial por completo.
+
+### División de Polinomios
+
+Para cada dos polinomios _f_ ( _x_ ) y _g_ ( _x_ ), existen dos polinomios _q_ ( _x_ ) y
+_r_ ( _x_) llamados el cociente y el resto de la división  _f_ ( _x_ ) por _g_ ( _x_ ). Satisfacen _f_ ( _x_ ) = _g_ ( _x_ ) \* _q_ ( _x_ ) + _r_ ( _x_ ) y el grado de  _r_ ( _x_ ) es menor que el grado de _g_ ( _x_ ).
+
+Por ejemplo, si _f_ ( _x_ ) = _x_<sup>3</sup> + _x_ + 1 and _g_ ( _x_ ) = _x_<sup>2</sup> + 1 entonces
+_q_ ( _x_ ) = _x_ y _r_ ( _x_ ) = 1. De hecho, _x_<sup>3</sup> + _x_ + 1 = ( _x_<sup>2</sup> + 1 )
+\* _x_ + 1.
+
+### Raíces de Polinomios
+
+Cuando un polinomio satisface _f_ (_a_) = 0 para algún valor específico _a_ (decimos que _a_ es una raíz de _f_), entonces no tenemos un residuo (_r_ ( _x_ ) = 0) al dividirlo por  (_x_ - _a_), por lo que podemos escribir _f_ (_x_) = (_x_ - _a_) \* _q_ ( _x_ ), y deg(_q_) = deg(_f_) - 1. Un hecho similar es cierto para _k_ raíces. Es decir, si _a_<sub>i</sub> es una raíz de _f_ para todos los _i_ = 1, 2, …, _k_, entonces existe un polinomio _q_ de grado deg(_f_) - _k_ para el cual  _f_ ( _x_ ) = ( _x_ - _a_<sub>1</sub> )( _x_ -
+_a_<sub>2</sub> ) … ( _x_ - _a_<sub>_k_</sub> ) \* _q_ ( _x_ ) .
+
+### ¿Quieres saber más?
```

Comment:
```suggestion
### ¿Quiere saber más?
```

Reply:
Thanks for the RAE recommendations and for the review, also change the name from ES README.md to README.es.md for the following country translations to have a good structure.